src/pages/styles.less

```diff
@@ -118,13 +117,12 @@
 
 @media (min-width: 768px) {
   .jsonPretty {
-    height: 600px;
+    height: 440px;
```

Comment:
Why did the height of the json reader change? I think this line can be reverted

Reply:
Yeah it was changed to fit  the new design, the json side and the input field section are now same height.